connectors/google-pubsub-sink/v2.8.1.29/google-pubsub-sink.md

```diff
@@ -0,0 +1,377 @@
+---
+description: The Google Cloud Pub/Sub sink connector pulls data from Pulsar topics and persists data to Google Cloud Pub/Sub.
+author: ["StreamNative"]
+contributors: ["StreamNative"]
+language: Java
+document: 
+source: "https://github.com/streamnative/pulsar-io-google-pubsub/tree/v2.8.1.29/src/main/java/org/apache/pulsar/ecosystem/io/pubsub"
+license: Apache License 2.0
+tags: ["Pulsar IO", "Pub/Sub", "Sink"]
+alias: Google Cloud Pub/Sub Sink
+features: ["Use Google Cloud Pub/Sub sink connector to sync data from Pulsar"]
+license_link: "https://www.apache.org/licenses/LICENSE-2.0"
+icon: "/images/connectors/google-pubsub-logo.png"
+download: "https://github.com/streamnative/pulsar-io-google-pubsub/releases/download/v2.8.1.29/pulsar-io-google-pubsub-2.8.1.29.nar"
+support: StreamNative
+support_link: https://streamnative.io
+support_img: "/images/connectors/streamnative.png"
+dockerfile: "https://hub.docker.com/r/streamnative/pulsar-io-google-pubsub"
+owner_name: "StreamNative"
+owner_img: "/images/streamnative.png" 
+id: "google-pubsub-sink"
+---
+
+The [Google Cloud Pub/Sub](https://cloud.google.com/pubsub) sink connector pulls data from Pulsar topics and persists data to Google Cloud Pub/Sub topics.
+
+![](/images/connectors/google-pubsub-sink.png)
+
+# How to get
+
+::: tabs
+
+@@@ Function Worker
+
+You can get the Google Cloud Pub/Sub sink connector using one of the following methods if you use [Pulsar Function Worker](https://pulsar.apache.org/docs/en/functions-worker/) to run connectors in a cluster.
+
+- Download the NAR package from [the download page](https://github.com/streamnative/pulsar-io-google-pubsub/releases/download/v{{connector:version}}/pulsar-io-google-pubsub-{{connector:version}}.nar).
+
+- Build it from the source code.
+
+To build the Google Cloud Pub/Sub sink connector from the source code, follow these steps.
+
+  1. Clone the source code to your machine.
+
+     ```bash
+     git clone https://github.com/streamnative/pulsar-io-google-pubsub
+     ```
+
+  2. Build the connector in the `pulsar-io-google-pubsub` directory.
+     
+     ```bash
+     mvn clean install -DskipTests
+     ```
+
+     After the connector is successfully built, a `NAR` package is generated under the target directory. 
+
+     ```bash
+     ls target
+     pulsar-io-google-pubsub-{{connector:version}}.nar
+     ```
+
+@@@
+
+@@@ Function Mesh
+
+You can pull the Google Cloud Pub/Sub sink connector Docker image from the [Docker Hub](https://hub.docker.com/r/streamnative/pulsar-io-google-pubsub) if you use [Function Mesh](https://functionmesh.io/docs/connectors/run-connector) to run the connector.
+
+@@@
+
+:::
```

Comment:
This is not shown in the preview
![image](https://user-images.githubusercontent.com/50226895/147797574-89f116b7-208e-440f-b1cf-3cb5f82d52d7.png)